magento/magento-cloud-patches
# Make MC-43048 available for 2.4.3-p1

The rate limit patch is still needed in the latest Magento version 2.4.3-p1, but was not listed anymore for that version.


### Manual testing scenarios
1. Locally install Magento Cloud version 2.4.3-p1
2. See if the patch is listed with `vendor/bin/ece-patches status`
3. Add the patch in .magento.env.yaml 
4. Apply the patch with `vendor/bin/ece-patches apply`



### Contribution checklist
